am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestConstants.java

```diff
@@ -23,6 +23,7 @@
     public static final String OTHER_ACCESSOR_ID = "b";
     public static final Set<Permission> EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS =
         Stream.of(CREATE, READ, UPDATE).collect(toSet());
+    public static final String ATTRIBUTE = "/test";
```

Comment:
I am not sure that should be here. Attributes are at core of access management. Hiding it here among less important constants might be bad idea. It might make tests less readable.

Reply:
I removed it in recent code 😄